translations/en.json

```diff
@@ -752,7 +759,7 @@
     "your-contact-code": "Granting access authorizes this DApp to retrieve your contact code",
     "password-placeholder": "At least 6 characters",
     "clear-history-action": "Clear",
-    "version": "Version {{version}}",
+    "version": "Version",
```

Comment:
that's potentially a good idea but isn't it breaking the other languages?

Reply:
reason is, "version" the title, and actual "version number" are decoupled from each other in the figma design now.

https://www.figma.com/file/bPS9GrvMr7LnH7vnkmuveQfd/Settings?node-id=576%3A4963

Reply:
Yeah I understand I don't mind, I just think you need to fix the other language files because they will still require the {{version}} if you don't change them so it might have some consequences

Reply:
sorry, I didn't get it at first :) I do now. Will fix it.

In wider context, I'm getting close.

Because this PR is so minutially involved, just have to be extra extra thorough as we possibly can

Reply:
done